docs/development/testguide.rst

```diff
@@ -341,8 +341,8 @@ location, or as a *decorator* that takes effect for an entire test function
 (not including setup or teardown, which would have to be decorated separately).
 
 Furthermore, it is possible to change the location of the cache directory
-for the duration of the test run by setting the ``XDG_CACHE_HOME``
-environment variable.
+for the duration of the test run via environment variables, see
+:ref:`environment_variables`.
```

Comment:
That's not how it works.

Reply:
I'm not 100% I'm following. This section is about the test runner. Are you hinting at how `conftest.py` obliterates user-set `XDG_CACHE_HOME` values ?

Reply:
> This section is about the test runner.

Irrelevant.

> Are you hinting at how `conftest.py` obliterates user-set `XDG_CACHE_HOME` values ?

Yes.

Reply:
The paragraph is indeed true. I really should have known better because I am the author of the commit that (unintentionally) made our test runner behave like that: 433a534fed44d7c520c0648496c2c24bfdcc5ea7